app/editor/src/features/reports/av-overview/AVOverview.tsx

```diff
@@ -76,7 +76,9 @@ export const AVOverview: React.FC = () => {
       <FormikForm
         initialValues={instance}
         onSubmit={(values, { setSubmitting }) => {
-          handleSubmit(values);
+          handleSubmit(values)
+            .catch((err) => toast.warning(() => 'Failed to save.'))
+            .finally(() => toast.success(() => 'Evening overview saved.'));
```

Comment:
This is incorrect.  Think about what `finally` means

Reply:
@Fosol - closin this since I see you fixed it